database/src/main/postgres/runs/V1.9.7__create_partitioning.sql

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+CREATE OR REPLACE FUNCTION runs.create_partitioning(
+    IN  i_partitioning              JSONB,
+    IN  i_by_user                   TEXT,
+    IN  i_parent_partitioning_id    BIGINT = NULL,
+    OUT status                      INTEGER,
+    OUT status_text                 TEXT,
+    OUT id_partitioning             BIGINT
+) RETURNS record AS
+$$
+-------------------------------------------------------------------------------
+--
+-- Function: runs.create_partitioning(3)
+--      Creates a partitioning entry
+--
+-- Parameters:
+--      i_partitioning              - partitioning to create or which existence to check
+--      i_by_user                   - user behind the change
+--      i_parent_partitioning_id    - (optional) parent partitioning id
```

Comment:
```suggestion
--      i_parent_partitioning_id    - (optional) parent partitioning id, must already exist if specified 
```

or something like that